packages/templates/upload-pdf-to-google-drive/config.yml

```diff
@@ -1,26 +1,31 @@
 version: "0.2"
 addresses:
-  $username@mailscript.com:
+  $username@mailscript.io:
     keys:
       - name: owner
         read: true
         write: true
 workflows:
   - name: save-pdf-to-google-drive
-    input: $username@mailscript.com
+    input: $username@mailscript.io
+    trigger: attachments-trigger
     action: save-pdf-to-google-drive
+triggers:
+  - name: attachments-trigger
+    composition:
+      - criteria:
+          hasAttachments: true
 actions:
   - name: save-pdf-to-google-drive
     type: webhook
     config:
       body: |
         {
           "attachments": "{{msg.attachments}}",
-          "driveStoragePath": "path-to-save-pdfs-in-for-google-drive",
-          "credsClientId": "service-account-id",
-          "credsPrivateKey": "service-account-private-key"
+          "driveStoragePath": "path/to/save/pdfs/in/for/google/drive",
+          "googleDriveAuth": "{{integrations.google}}"
         }
-      url: "https://write-to-csv-endpoint.url"
+      url: "http://localhost:5001/mailscript-firebase/us-central1/googleDrivePdfUploader"
```

Comment:
And similarly here. I have pushed the cloud functions so it should be available.